srcpkgs/foliate/template

```diff
@@ -12,3 +12,7 @@ license="GPL-3.0-or-later"
 homepage="https://johnfactotum.github.io/foliate/"
 distfiles="https://github.com/johnfactotum/foliate/archive/${version}.tar.gz"
 checksum=e3f7ab5242aa6d936b66d6a41147bf17c0334d7675b5e38e64829d204fd3712a
+
+post_install() {
+	ln -sv /usr/bin/com.github.johnfactotum.Foliate ${DESTDIR}/usr/bin/foliate
```

Comment:
I prefer relative symlink for this one, looks better when inspecting filesystem from other system.

    ln -s com.github.johnfactotum.Foliate ${DESTDIR}/usr/bin/foliate

Reply:
I agree.